src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -1034,27 +1030,15 @@ export class EditDataGridPanel extends GridParentComponent {
 		// handleInitializeTable() will be called *after* the first time handleChanges() is called
 		// so, grid must be there already
 
-		if (this.dataSet.dataRows && this.dataSet.dataRows.getLength() > 0) {
+		if (this.placeHolderDataSets[0].dataRows && this.placeHolderDataSets[0].dataRows.getLength() > 0) {
 			this.table.grid.scrollRowToTop(0);
 		}
 
-		if (this.dataSet.resized) {
-			// Re-rendering the grid is expensive. Throttle so we only do so every 100ms.
-			this.dataSet.resized.throttleTime(100)
```

Comment:
I don't see any `throttleTime` equivalent in the changed code (unless I'm missing something).  Is the throttling no longer needed to avoid excessive re-rendering?

Reply:
Throttling is no longer needed to avoid re-rendering as its only rerendered when its needed